admin-ui/plugins/auth-server/redux/features/scriptSlice.js

```diff
@@ -0,0 +1,59 @@
+import reducerRegistry from 'Redux/reducers/ReducerRegistry'
+import { createSlice } from '@reduxjs/toolkit'
+
+const initialState = {
+  scripts: [],
+  script: {},
+  loading: true,
+  scriptTypes: [],
+}
```

Comment:
Consider initial loading=false

Initialize loading to false; set true only when fetching begins.